src/apps/Console/ConsoleFeedback.js

```diff
@@ -216,7 +216,38 @@ export default function ConsoleFeedback({
         </div>
       </>
     )
-  } else if (currentParsedCommand[0] === 'act') {
+  } else if (
+    currentParsedCommand[0] === 'act' &&
+    currentParsedCommand.length <= 2
+  ) {
+    const agentInstalled =
+      apps.filter(app => app.name.toLowerCase() === 'agent').length > 0
+    if (!agentInstalled) {
+      return (
+        <>
+          <p
+            css={`
+            ${textStyle('body2')}
+            margin: 0 0 ${GU}px 0;
+          `}
+          >
+            There are no agent instances installed in this organization. Please
+            read{' '}
+            <Link
+              external
+              href="https://hack.aragon.org/docs/cli-dao-commands#dao-install"
+            >
+              the documentation
+            </Link>{' '}
+            on how to install an application using the CLI, or go to our{' '}
+            <Link external href="https://discord.gg/ruBR6GN">
+              Discord
+            </Link>{' '}
+            server to help you install it.
```

Comment:
Let's prefer our spectrum link instead, since that is where we should route developer support queries.